sdk-tests/src/test/java/io/dapr/it/methodinvoke/grpc/MethodInvokeIT.java

```diff
@@ -88,11 +88,12 @@ public void testInvokeTimeout() throws Exception {
             MethodInvokeServiceGrpc.MethodInvokeServiceBlockingStub stub = createGrpcStub(client);
             long started = System.currentTimeMillis();
             SleepRequest req = SleepRequest.newBuilder().setSeconds(1).build();
-            String message = assertThrows(StatusRuntimeException.class, () -> stub.sleep(req)).getMessage();
+            StatusRuntimeException exception = assertThrows(StatusRuntimeException.class, () -> stub.sleep(req));
             long delay = System.currentTimeMillis() - started;
+            Status.Code code = exception.getStatus().getCode();
+
             assertTrue(delay >= TIMEOUT_MS, "Delay: " + delay + " is not greater than timeout: " + TIMEOUT_MS);
-            assertTrue(message.contains("DEADLINE_EXCEEDED"), "The message contains DEADLINE_EXCEEDED: " + message);
```

Comment:
This was working before, why removed?

Reply:
This is related to this -> https://github.com/dapr/java-sdk/pull/1089#issuecomment-2312502731 That error message if flaky. I don't think we should remove the assertion, but we need to find why Dapr Runtime is returning two different messages, I am not sure to understand the difference between them.

Reply:
@artursouza I have seen this IT test failing on and off on multiple PRs, just as @salaboy mentioned. In my opinion, relying on exception message leads to flaky tests. In this particular test we are interested in the fact that a timeout has been reached, hence looking for `Status.Code` equal to `DEADLINE_EXCEEDED` should be enough.